testing/esp/esdlcmd/baselines/wstest-cpp-installdir/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Copyright and license info? Comment applies to all new files, with the possible exception of the readme.

Reply:
These are auto-generated by the esdl tool, and are added here not as code but as baseline output for the test cases. If I add copyright comments then I'd need to take that into account when making the comparison against the test case output. I'd like to avoid that if I can but if the comments are required even this situation I'll do that.